spec/index.bs

```diff
@@ -1187,6 +1187,23 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         is {{GPUBufferBindingType/"storage"}}
         or {{GPUBufferBindingType/"read-only-storage"}}.
 
+    <tr><td><dfn>minUniformBufferOffsetAlignment</dfn>
+        <td>{{GPUSize32}} <td>Lower <td>256
+    <tr class=row-continuation><td colspan=4>
+        The required alignment for {{GPUBufferBinding/offset}} for bindings with a
+        {{GPUBindGroupLayoutEntry}} |entry| for which
+        |entry|.{{GPUBindGroupLayoutEntry/buffer}}?.{{GPUBufferBindingLayout/type}}
```

Comment:
What about dynamic offsets? It seems to me that the limit should apply to them as well.